scripts/startLocalFunctionsTestnet.js

```diff
@@ -0,0 +1,54 @@
+const process = require("process")
+const path = require("path")
+const fs = require("fs")
+const { startLocalFunctionsTestnet } = require("@chainlink/functions-toolkit")
+const { utils, Wallet } = require("ethers")
+// Loads environment variables from .env.enc file (if it exists)
+require("@chainlink/env-enc").config("../.env.enc")
+;(async () => {
```

Comment:
nit: there is a semi colon here at the start?

Reply:
It is a JavaScript requirement for the edge case have a require statement before a `(async () => { ... })()` statement.

There are very few cases where semicolons are actually required in JS, but this is one of them.